flytekit/core/resources.py

```diff
@@ -1,5 +1,7 @@
 from dataclasses import dataclass
-from typing import Optional
+from typing import List, Optional
+
+from flytekit.models import task as _task_models
```

Comment:
let's avoid the _ syntax.  can we just do
```suggestion
from flytekit.models import task as task_models
```